src/software/thunderscope/gl/layers/gl_world_layer.py

```diff
@@ -361,12 +368,16 @@ def _update_robots_graphics(self) -> None:
             friendly_colour,
             self.friendly_robot_graphics,
             self.friendly_robot_id_graphics,
+            self.friendly_robot_name_graphics,
+            ROBOT_NAMES_FROM_ID,
         )
         self.__update_robot_graphics(
             self._cached_enemy_team,
             enemy_colour,
             self.enemy_robot_graphics,
             self.enemy_robot_id_graphics,
+            self.enemy_robot_name_graphics,
+            {},
```

Comment:
Why not also show enemy robot name?

Reply:
I guess it is okay.

Reply:
We don't have robot names yet, and I didn't bother defining a constant for them. You can key whatever names you like to each id of the enemy robots, but I figured that doing so would only clutter the screen unnecessarily, especially if I'm making random names.

Reply:
Yichen, our ex team captain, wants to setup a field for 4v4 or something along the line. What happens is that sometime the enemy is one of our robot that has a same name, and it might be nice to show that. 

I guess it is ok to leave it like this.

Reply:
Oh, right now the issue is that the enemy robots have hardcoded ids from 1-6, so if I use the existing constant to give those robots the label then we get duplicates.
I'd have to change the ids or something, but we don't have enough for 12 robots I think.